arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json

```diff
@@ -1580,6 +1581,16 @@
               "type": "string",
               "format": "date-time",
               "description": "the date/time of the last published gateway"
+            },
+            "installerDownload": {
+              "type": "string",
+              "readOnly": true,
```

Comment:
These are 'readonly' in the sense that service set their values and cannot be modified by the client. These are set when the server sends the response. I wasn't aware that they would be stripped during serialization ,thanks for pointing that out.